source/Camera/InfoUtilOther/C++Halcon/HalconConnectUsingZivid/HalconConnectUsingZivid.cpp

```diff
@@ -0,0 +1,103 @@
+#include <Zivid/Zivid.h>
+#include <halconcpp/HalconCpp.h>
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <chrono>
+#include <iostream>
+
+
+void savePointCloud(const HalconCpp::HObjectModel3D &model, const std::string filename)
+{
+	const HalconCpp::HString filetype{ "ply" };
+	const HalconCpp::HString filenameCstring{ filename.c_str() };
+	const HalconCpp::HString genParamName{ "invert_normals" };
+	const HalconCpp::HString genParamValue{ "false" };
+	model.WriteObjectModel3d(filetype, filenameCstring, genParamName, genParamValue);
+}
+
+
+HalconCpp::HObjectModel3D toHalconPointCloud(const Zivid::PointCloud &pc)
```

Comment:
I would maybe rename this to zividToHalcon or zividToHalconPointCloud. with just toHalconPointCloud it may not be clear from a first glance what is it converted form. One has to look into the argument type. Maybe it is ok and I am crazy about details. You decide.